nautobot/extras/templatetags/job_buttons.py

```diff
@@ -29,6 +29,7 @@
 <input type="hidden" name="object_model_name" value="{object_model_name}">
 <input type="hidden" name="_schedule_type" value="immediately">
 <input type="hidden" name="_return_url" value="{redirect_path}">
+<input type="hidden" name="_commit" value="on">
```

Comment:
This fix line isn't needed in 2.x, but the rest of this PR is, correct?

Reply:
Correct, but I forgot sites go away in 2.0, so I will have to change the tests.